README.md

```diff
@@ -9,36 +9,60 @@
 ![](https://img.shields.io/github/languages/code-size/peak-ai/platform-global-images)
 ![](https://img.shields.io/github/repo-size/peak-ai/platform-global-images)
 
-This repository contains the source code for all the global images available on Peak Platform.
+This repository contains the source code for the global images that are available on Peak.<p>
+For more information about using Docker images with Peak, see [Image Management](https://platformsupport.peak.ai/support/solutions/folders/80000683574) in the Peak knowledge base.
 
-## Available Images
+## Getting started
+### Directory structure
+The images contained in this repository are organised in the following way:
+> peak-ai/platform-global-images/platform function/language/image-package/version
```

Comment:
would be great if it can be shown in a tree structure.

Reply:
I will have a go at doing this.

Reply:
Updated.